include/common.sh

```diff
@@ -45,6 +45,39 @@ addCleanupTrapCmd(){
   [ -n "${xtrc}" ] && set -x
 }
 
+# Temporary files
+for tmp in "${TMPDIR}" "${TMP}" /tmp; do
```

Comment:
Can you use a different var so that it doesnt collide with locals, maybe ZOPEN_TMP_DIR?